packages/backend/src/services/flow.ts

```diff
@@ -31,6 +33,8 @@ export const processFlow = async (options: ProcessFlowOptions) => {
     }
   } catch (error) {
     if (error instanceof EarlyExitError === false) {
+      // this should only send error email when scheduler has issues during actual runs
+      await sendErrorEmail(flow, $.userEmail, options.testRun)
```

Comment:
will this definitely only be executed during actual flows

Reply:
cus i set the testRun so it shouldnt send even if its not actual flow but I think I should catch in the worker like what u said